docroot/modules/humsci/hs_blocks/hs_blocks.module

```diff
@@ -34,3 +34,17 @@ function hs_blocks_theme($existing, $type, $theme, $path) {
     ],
   ];
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function hs_blocks_preprocess_block__block_content(&$variables) {
```

Comment:
Double underscore?

Reply:
you can target specific hook ids to be more specific in the preprocess. `preprocess_block__BLOCK_ID`